modules/consensus.go

```diff
@@ -110,6 +110,10 @@ type (
 		// SiafundPoolDiffs are the siafund pool diffs that were applied to the
 		// consensus set in the recent change.
 		SiafundPoolDiffs []SiafundPoolDiff
+
+		// Synced indicates whether or not the ConsensusSet is synced with its
+		// peers.
+		Synced bool
```

Comment:
Just confirmed with some testing, doesn't screw up persistence stuff. Looked at the code as well.